pkg/cli/webhook/secret.go

```diff
@@ -20,11 +19,10 @@ const (
 	gitlabWebhookSecretName = "gitlab-webhook-secret"
 )
 
-func (w *Options) createWebhookSecret(ctx context.Context, provider string, response *response) error {
-	secretName := getSecretName(provider)
+func (w *Options) createWebhookSecret(ctx context.Context, response *response) error {
 	_, err := w.Run.Clients.Kube.CoreV1().Secrets(w.RepositoryNamespace).Create(ctx, &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: secretName,
+			Name: w.RepositoryName,
```

Comment:
I was wondering if maybe we should have org/repo in there but that may be a lenght too big for kubernetes secret name length (i think 256 max)

but user can have multiple repo name to different org/user

ie: github.com/chmouel/pipelines github.com/shivam/pipelines

which would confilict.....

but let's maybe address this in a later issue if you like

Reply:
yeah we create repo cr with org-repo right?
so we use the same for secret

Reply:
ah right so either way we may have an issue with the 256 field :-D 

(we maybe need to trim)